### PR TITLE
Add page for Audit Token Right Adjusted

### LINKED ISSUE #2465: Detailed Tracking - Audit Token Right Adjusted
Win10 includes the Audit Token Right Adjusted subcategory in Detailed Tracking. Please update this page/create a subpage for Audit Token Right Adjusted.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 739bc932-12f0-9595-fbd2-d5b0df73fe93
* Version Independent ID: cd1f89ad-34db-bb96-e9ef-2137958c9403
* Content: [Advanced security audit policy settings (Windows 10)](https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/advanced-security-audit-policy-settings#detailed-tracking)
* Content Source: [windows/security/threat-protection/auditing/advanced-security-audit-policy-settings.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/threat-protection/auditing/advanced-security-audit-policy-settings.md)
* Product: **w10**
* GitHub Login: @brianlic-msft
* Microsoft Alias: **justinha**

### PR DESCRIPTION
**Pull request by tom-henderson**
Fixes #2465.

The events list may not be complete.

### CODE REVIEW

**Comment by e0i** [comment]
cc @mypil

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #2465 

Thank you.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by illfated** [comment]
Rich Text diff view:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4822/files?short_path=e195541#diff-e19554129fbabb6686f2db8ba86bbc25 (`advanced-security-audit-policy-settings.md`)
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4822/files?short_path=7384bba#diff-7384bba5d82f9a90a54d1fcab927a8d8 (`audit-token-right-adjusted.md`)

I have not done any proofreading yet.

**Comment by illfated** [comment]
Still misspelled, "priveleges" should be **privileges** to be a correct English word (notice the use of 2 i-s and 2 e-s in the correct spelling).

**Comment by tom-henderson** [comment]
Thanks @illfated, I've updated this with the correct spelling.

File: windows/security/threat-protection/auditing/advanced-security-audit-policy-settings.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Detailed Tracking security policy settings and audit events can be used to monit
 - [Audit Process Termination](audit-process-termination.md)
 - [Audit RPC Events](audit-rpc-events.md)
 - [Audit Credential Validation](https://docs.microsoft.com/windows/security/threat-protection/auditing/audit-credential-validation)
-  > **Note:**  For more information, see [Security Monitoring](https://blogs.technet.microsoft.com/nathangau/2018/01/25/security-monitoring-a-possible-new-way-to-detect-privilege-escalation/)
+- [Audit Token Right Adjusted](https://docs.microsoft.com/windows/security/threat-protection/auditing/audit-token-right-adjusted)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/advanced-security-audit-policy-settings.md]:
```suggestion
- [Audit Token Right Adjusted](https://docs.microsoft.com/windows/security/threat-protection/auditing/audit-token-right-adjusted).
```

File: windows/security/threat-protection/auditing/audit-token-right-adjusted.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Audit Token Right Adjusted
+
+**Applies to**
+-   Windows 10
+-   Windows Server 2016
+
+
+Audit Token Right Adjusted allows you to audit events generated by adjusting the privileges of a token. 
+
+For more information, see [Security Monitoring: A Possible New Way to Detect Privilege Escalation](https://blogs.technet.microsoft.com/nathangau/2018/01/25/security-monitoring-a-possible-new-way-to-detect-privilege-escalation/)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/audit-token-right-adjusted.md]:
```suggestion
For more information, see [Security Monitoring: A Possible New Way to Detect Privilege Escalation](https://blogs.technet.microsoft.com/nathangau/2018/01/25/security-monitoring-a-possible-new-way-to-detect-privilege-escalation/).
```

**Comment by mypil** [comment]
@tom-henderson - please merge the commit on the copy/edit review made by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

File: windows/security/threat-protection/auditing/advanced-security-audit-policy-settings.md (hunk 1 above)
Comment (illfated) [reply windows/security/threat-protection/auditing/advanced-security-audit-policy-settings.md]: Maybe, but I wonder if the period might look a bit out of place after a link,
which is only supposed to be an entry in a sort of table. The link entries
don't look like regular sentences to me. Myself, I would have skipped the period
 and dismissed or marked this comment as resolved if I wanted to leave the text as it already is.

Comment (tom-henderson) [reply windows/security/threat-protection/auditing/advanced-security-audit-policy-settings.md]: I agree I don't think the period belongs here. This is consistent with the other items in this list.

**Comment by tom-henderson** [comment]
@mypil - copy review comments addressed, thanks.

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #2465.

Thank you.

**Pull request merged**